### PR TITLE
New plate/ab 384plate

### PR DESCRIPTION
**Pull request by CharlotteWard**
applied biosystems microamp optical 384 well reaction plate to plate library

### CODE REVIEW

File: inventory/testinventory/make_plate_library.go [hunk 1]
```diff
@@ -893,47 +895,44 @@ func make96DeepWellLowVolumePlate() *wtype.LHPlate {
...
+	xdim := 3.17 // G1: diameter at top of well
+	ydim := 3.17 // G1: diameter at top of well
+	zdim := 9.09 // L: depth of well from top to bottom
 
-	bottomh := 0.0 // N: Used to model different well bottom to body shape. This is the height of the bottom part.
+	bottomh := 0.61 // N: bottom of well to resting plane
 
-	minVolume := 15.0
-	maxVolume := 65.0 // well capacity is therefore 50.0
+	minVolume := 0.5
```
Comment (chr1sgrant) [inline inventory/testinventory/make_plate_library.go]: Is the minimal volume really this low?

Have you tested this?

File: inventory/testinventory/make_plate_library.go [hunk 2]
```diff
@@ -893,47 +895,44 @@ func make96DeepWellLowVolumePlate() *wtype.LHPlate {
 	return plate
 }
 
-// make plate Labcyte384PPStdV
-// Lot number 04090140
-// Part number P-05525
-// Specs retrieved from
-// https://www.labcyte.com/media/pdf/SPC-Qualified-Microplate-384PP.pdf
-func makeLabcyte384PPStdV() *wtype.LHPlate {
```
Comment (chr1sgrant) [inline inventory/testinventory/make_plate_library.go]: Is it intended to delete this previous plate?

Comment (chr1sgrant) [reply inventory/testinventory/make_plate_library.go]: `makeLabcyte384PPStdV`

File: inventory/testinventory/make_plate_library.go (hunk 1 above)
Comment (CharlotteWard) [reply inventory/testinventory/make_plate_library.go]: Will check this with Xander.

File: inventory/testinventory/make_plate_library.go (hunk 2 above)
Comment (CharlotteWard) [reply inventory/testinventory/make_plate_library.go]: Ooops, no, good spot.

**Pull request closed without merging**